doc/overview.md

```diff
@@ -0,0 +1,142 @@
+# APIcast Overview
+
+APIcast is an NGINX based API gateway used to integrate your internal and external API services with 3scale’s API Management Platform.
+
+Here you’ll learn more about deployment options, environments provided, and how to get started.
+
+## Deployment options
+
+You can use APIcast hosted or self-managed, in both cases, it needs connection to the rest of the 3scale API management platform:
+- **<a href="hosted">APIcast hosted</a>**: 3scale hosts the gateway in the cloud. The API gateway is already deployed for you and it's limited to 50,000 calls per day.

```

Comment:
this links points nowhere, well, to file `hosted` in the same folder as this document.